implementation/src/main/java/com/teampc/controller/grading/SubmissionResultsController.java

```diff
@@ -0,0 +1,130 @@
+package com.teampc.controller.grading;
+
+import com.teampc.controller.questionview.QuestionViewController;
+import com.teampc.dao.SubmissionDAO;
+import com.teampc.model.admin.access.UserSession;
+import com.teampc.model.question.Question;
+import com.teampc.model.testtaking.QuestionResponse;
+import com.teampc.model.testtaking.Submission;
+import com.teampc.utils.FXUtils;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.Pane;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ResourceBundle;
+
+/**
+ * Created by zarend on 12/8/15.
+ */
+public class SubmissionResultsController implements Initializable {
+   private static final Logger LOG = LoggerFactory.getLogger(SubmissionResultsController.class);
+
+   private QuestionViewController currentQuestionController;
+
+   private Submission submission;
+
+   private boolean isGrading;
+
+   private int currQuestionIdx = 0; // TODO: prevent this from going less than 0 or more than numQuestions - 1
```

Comment:
I planned on fixing the Next and Prev buttons so they don't go out of bounds after I get this merged